pages/api/prompt.js

```diff
@@ -0,0 +1,14 @@
+import { Configuration, OpenAIApi } from "openai";
+
+const configuration = new Configuration({
+  apiKey: process.env.OPEN_AI_KEY,
+});
+const openai = new OpenAIApi(configuration);
+
+export default async function handler(req, res) {
+  const completion = await openai.createCompletion({
+    model: "text-davinci-003",
+    prompt: req.body,
+  });
+  res.status(200).json({ text: completion.data.choices[0].text });
+}
```

Comment:
1) just keep things typescript, there's no benefit to mixing js and ts files -- makes things worse

2) did you try the `app` dir `route.ts` file?

Reply:
Don't think the route handlers on `app` dir are all that ready yet -- this is good but should be TS